internal/testutil/acc/cluster.go

```diff
@@ -49,14 +50,22 @@ func GetClusterInfo(tb testing.TB, req *ClusterRequest) ClusterInfo {
 	if req.Geosharded {
 		clusterTypeStr = "GEOSHARDED"
 	}
+	dependsOnClause := ""
```

Comment:
knit: we could also do extraConfig := req.ExtraConfig,   and add it depends info if needed so no need to pass a new param to clusterTerraformStr